python/packages/nisar/products/insar/InSAR_base_writer.py

```diff
@@ -361,12 +361,21 @@ def add_RSLC_to_procinfo_params_group(self, rslc_name: str):
                                   ' composite of data collected in multiple'
                                   ' radar modes, "False" otherwise')
         ds_params = [
+            DatasetParams(
+                "rfiMitigation",
+                rfi_mitigation,
+                (
+                    f'Algorithm used for radio frequency interference (RFI) mitigation in ' \
+                      'the {rslc_name} RSLC, either "ST-EVD" or "FDNF" (or "disabled" if no RFI ' \
+                      'mitigation was applied)'
+                ),
+            ),
             DatasetParams(
                 "rfiMitigationApplied",
                 rfi_mitigation_flag,
                 (
-                    "Flag to indicate if RFI mitigation has been applied"
-                    f" to {rslc_name} RSLC"
+                    "Flag to indicate if radio frequency interference (RFI) mitigation was applied"
```

Comment:
This is not of this PR, but I wanted to confirm that you don't need to convert the flag `rfi_mitigation_flag` above to a byte string: `np.bytes_()`

Reply:
Yes, that's unnecessary. Thank you, @xhuang-jpl and @nemo794 .